public/config/kc/keycloak.json

```diff
@@ -0,0 +1,8 @@
+{
+  "realm": "fcf0kpqr",
+  "auth-server-url": "https://dev.oidc.gov.bc.ca/auth/",
+  "ssl-required": "external",
+  "resource": "account-web",
```

Comment:
we will use a different client for this, but can change when we integrate staff login

Reply:
sure